server/config_test.go

```diff
@@ -0,0 +1,92 @@
+package server
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+)
+
+const (
+	fixturePath  = "fixtures/crio.conf"
+	temporalPath = "/tmp"
```

Comment:
You can drop this as well, and just use "" in the first argument to os.TempFile as /tmp is Linux specific but if you use "" (empty string) golang will choose the appropriate temp directory for a given os